torchao/quantization/quant_api.py

```diff
@@ -488,6 +489,17 @@ def int8_dynamic_activation_int8_semi_sparse_weight():
     """
     return int8_dynamic_activation_int8_weight(layout_type=SemiSparseLayoutType())
 
+def float8_weight_only(target_dtype: torch.dtype = torch.float8_e4m3fn):
+    """
+    Applies float8 weight-only symmetric per-channel quantization to linear layers.
+    """
+    def apply_float8wo_quant(weight):
+        # avoid circular dep
```

Comment:
maybe you want to import `to_affine_quantized_floatx` here, I'm also refactoring this file to change the import to the file to avoid circular dep as well